src/Payments.Mvc/Views/Teams/ConfirmAccount.cshtml

```diff
@@ -0,0 +1,129 @@
+                </dd>
+                <dt>
+                    @Html.DisplayNameFor(model => model.Account)
+                </dt>
+                <dd>
+                    @Html.DisplayFor(model => model.Account)
+                </dd>
+                <dt>
+                    @Html.DisplayNameFor(model => model.Object)
+                </dt>
+                <dd>
+                    @Html.DisplayFor(model => model.Object)
+                </dd>
+                <dt>
+                    @Html.DisplayNameFor(model => model.SubAccount)
+                </dt>
+                <dd>
+                    @Html.DisplayFor(model => model.SubAccount)
+                </dd>
+                <dt>
+                    @Html.DisplayNameFor(model => model.SubObject)
+                </dt>
+                <dd>
+                    @Html.DisplayFor(model => model.SubObject)
+                </dd>
+                <dt>
+                    @Html.DisplayNameFor(model => model.Project)
+                </dt>
+                <dd>
+                    @Html.DisplayFor(model => model.Project)
+                </dd>
+                <dt>
+                    @Html.DisplayNameFor(model => model.IsDefault)
+                </dt>
+                <dd>
+                    @Html.DisplayFor(model => model.IsDefault)
+                </dd>
+            </dl>
+            <input type="hidden" asp-for="Chart" />
+            <input type="hidden" asp-for="Account" />
+            <input type="hidden" asp-for="Object" />
+            <input type="hidden" asp-for="SubAccount" />
+            <input type="hidden" asp-for="SubObject" />
+            <input type="hidden" asp-for="Project" />
+            <input type="hidden" asp-for="IsDefault" />
+
+
+            <input type="hidden" id="confirm" name="confirm" value="true" />
+            <div class="form-group">
+                <input type="submit" value="Confirm" class="btn btn-default" />
+                <a class="btn btn-secondary" id="edit">Edit</a>
+            </div>
+        </form>
+    </div>
+</div>
+
+<div>
+    <a asp-action="Index">Back to List</a>
+</div>
+
+@section Scripts {
+    @{await Html.RenderPartialAsync("_ValidationScriptsPartial");}
+    <script type="text/javascript">
+
+        $('#edit').on('click',
+            function() {
+                $("#confirm").val("false");
+                $("#confirm-create").submit();
+            });
+
+
+    </script>
+
+    @*<script type="text/javascript">
+            $("#Account").on("blur",
+                function() {
+                    const account = $("#Account").val();
+                    const chart = $("#Chart").val();
+                    const subAcccount = $("#SubAccount").val();
+                    //fetch(`/financial/info?chart=4&account=CRU9033`, { credentials: "same-origin" })
+                    fetch(`/financial/info?chart=` + chart + `&account=`+ account +`&subAccount=` + subAcccount, { credentials: "same-origin" })

```

Comment:
@jSylvestre if you are using backticks then you shouldn't also be doing string contact, since you can just inline instead. ex ``` `/financial/info?chart=${chart}&account=....` ```